scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -56,6 +62,12 @@ task GatherVCFs {
     OUTPUT=${output_vcf_name}.vcf
   >>>
 
+  runtime {
+    docker: "$__docker__"
```

Comment:
As suggested by @davidbenjamin make docker an argument to the wdl. similarly for memory, disk.